enclave-tls/src/enclave_quotes/sgx-la/collect_evidence.c

```diff
@@ -10,6 +10,12 @@
 #include "sgx_stub_u.h"
 #include "sgx_la.h"
 
+/* A local attestation needs the interaction of two different enclaves. QE as
```

Comment:
Change to `The local attestation requires to exchange the target info between ISV enclaves as the prerequisite. This is out of scope in enclave-tls because it requires to establish a out of band channel to do that. Instead, introduce QE as the intermediator. One ISV enclave as attester can request the local reports signed by QE and the opposite end of ISV enclave as verifier can check the validation of local report through calling sgx_qe_get_quote() which verifies the signed local report. Once getting quote successfully, it presents ISV enclave's local report has been fully verified.`